sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -445,6 +445,10 @@ public String getVaultUrl() {
                         return Mono.defer(() -> Mono.just(new PollResponse<>(LongRunningOperationStatus.IN_PROGRESS,
                             pollingContext.getLatestResponse().getValue())));
                     }
+                    if (deletedCertificateResponse.getStatusCode() == HttpURLConnection.HTTP_FORBIDDEN) {
+                        return Mono.defer(() -> Mono.just(new PollResponse<>(LongRunningOperationStatus.fromString("FORBIDDEN", true),
```

Comment:
Based on the feedback from API review.
this is not supposed to be here.
This logic is supposed to be in the poll operation of beginCreateCertificate.

Reply:
As discussed offline, we should not be swallowing 403 on beginCreateCertificate but on beginRecover and beginDelete polling operations. And the poller should return a `SUCCESSFULLY_COMPLETED` poller operation.